tools/convert/ruleconvert_test.go

```diff
@@ -25,3 +28,44 @@ func Test_keysToLowercase(t *testing.T) {
 		})
 	}
 }
+
+func Test_ConvertTypes(t *testing.T) {
+	tests := []struct {
+		name string
+		m    map[string]any
+		want map[string]any
+	}{
+		{"clearfreq_int", map[string]any{
```

Comment:
Don't we need test cases for `int32` as well?

Reply:
As I understand it, it's not possible for YAML to represent an int32 in Go, so it's not necessary to test for it.